src/Microsoft.Management.Infrastructure.CimCmdlets/CimAsyncOperation.cs

```diff
@@ -33,7 +33,6 @@ public CimAsyncOperation()
         {
             this.moreActionEvent = new ManualResetEventSlim(false);
             this.actionQueue = new ConcurrentQueue<CimBaseAction>();
-            this._disposed = 0;
```

Comment:
I recommend to keep this. It's better to explicitly initialize the value even though that's the default value.